src/api_manager/service_control/proto.cc

```diff
@@ -799,6 +833,18 @@ const SupportedLabel supported_labels[] = {
 const int supported_labels_count =
     sizeof(supported_labels) / sizeof(supported_labels[0]);
 
+const SupportedLabel by_consumer_supported_labels[] = {
```

Comment:
we can do something in the set_xxx function.  to make sure only set to the second operation for the new label